docs/guides/cross-referencing-with-sphinx.rst

```diff
@@ -77,11 +77,6 @@ Another example, here we add a target to a paragraph:
 Then we can reference it using ```target to paragraph`_``,
 that will be rendered as: `target to paragraph`_.
 
-The reference displays the name of the target by default,
-but we can use any text you want. For example:
-```my custom text <target to paragraph>`_``,
-that will be rendered as: `my custom text <target to paragraph>`_.
```

Comment:
Why is this a lie? 🤔

Reply:
you can't do this, it isn't supported to set a custom text.

Reply:
yeah, but the generated link is wrong, it should be https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#target-to-paragraph

Reply:
Oh wow, that was subtle. So much reST to learn still...